DigitalExcellence/dex-frontend
# bugfix/426-Privacy-modal-responsive

## Description

Quick fix for the privacy modal, I also went ahead and updated the delete-generic-modal because it had the same issue.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Open an incognito tab
2. Go to the login page
3. Click FHICT and check if the privacy modal looks good on desktop and mobile

## Link to issue

Closes: #426
